.script/tests/KqlvalidationsTests/CustomFunctions/ASimDnsMicrosoftNXLog.json

```diff
@@ -0,0 +1,182 @@
+{
+  "FunctionName": "ASimDnsMicrosoftNXLog",
```

Comment:
@jszigetvari-nxlog we already have custom table here : https://github.com/Azure/Azure-Sentinel/blob/master/.script/tests/KqlvalidationsTests/CustomTables/NXLog_DNS_Server_CL.json

Can you remove this file too, we don't add tables in CustomFunction?

Reply:
Removed the file as you asked.